netcon/includes/mdllinit.h

```diff
@@ -286,7 +289,7 @@ DLLTracker_id = (char *)API.vp[1];
 DLLGame_is_master_tracker_game = API.vp[2];
 DLLGame_mode = *API.vp[3];
 // DLLCurrent_pilot = (pilot *)API.vp[4];
-DLLLocalD3Dir = (char *)(pilot *)API.vp[5];
+DLLBase_directories = (std::vector<std::filesystem::path> *)API.vp[5];
```

Comment:
This variable should not be exported and accessible directly to netcon modules.

Reply:
OK. I removed that variable.